src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs

```diff
@@ -0,0 +1,859 @@
+/*
+ * Copyright(c) 2019 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using Tizen.NUI.Binding;
+
+namespace Tizen.NUI.BaseComponents
+{
+    /// <summary>
+    /// Class with the text and style parameters to be rendered into a pixel buffer.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
```

Comment:
could you remove <since_tizen> 8 </since_tizen> ?
this tag would be add when ACR is completed and [EditorBrowsable(EditorBrowsableState.Never)] is removed.